packages/amplify-provider-awscloudformation/src/system-config-manager.js

```diff
@@ -114,25 +116,29 @@ async function getRoleCredentials(context, profileName, profileConfig) {
       context.print.info(`  ${profileConfig.mfa_serial}`);
       mfaTokenCode = await getMfaTokenCode();
     }
-
+    logger('getRoleCredentials.aws.STS', [sourceProfileAwsConfig])();
     const sts = new aws.STS(sourceProfileAwsConfig);
-    const roleData = await sts
-      .assumeRole({
-        RoleArn: profileConfig.role_arn,
-        RoleSessionName: roleSessionName,
-        DurationSeconds: profileConfig.duration_seconds,
-        ExternalId: profileConfig.external_id,
-        SerialNumber: profileConfig.mfa_serial,
-        TokenCode: mfaTokenCode,
-      })
-      .promise();
-
-    roleCredentials = {
-      accessKeyId: roleData.Credentials.AccessKeyId,
-      secretAccessKey: roleData.Credentials.SecretAccessKey,
-      sessionToken: roleData.Credentials.SessionToken,
-      expiration: roleData.Credentials.Expiration,
+    const assumeRoleRequest = {
+      RoleArn: profileConfig.role_arn,
+      RoleSessionName: roleSessionName,
+      DurationSeconds: profileConfig.duration_seconds,
+      ExternalId: profileConfig.external_id,
+      SerialNumber: profileConfig.mfa_serial,
+      TokenCode: mfaTokenCode,
     };
+    const log = logger('getRoleCredentials.sts.assumeRole', [assumeRoleRequest]);
```

Comment:
This seems somewhat sensitive as well. Is it part of the redacted content?

Reply:
Yeah I saw the arguments it uses and those values will be redacted